pootle/settings/30-site.conf

```diff
@@ -15,6 +15,8 @@ MANAGERS = ADMINS
 
 DEBUG = False
 
+CANONICAL_URL = ""
```

Comment:
If we're expecting people to set this or use Sites.  Then this definitely needs to be in 90-local.conf.sample.  And I'm guessing now that we might want to to be localhost if we want Pootle to be able to run straight away after init?

We moved all our settings to the POOTLE_ namespace just to make sure its clear what we manage and what we don't.  Unless we're using something from Django I'd say its our thing and prevents any future conflict when Django decided CANONICAL_URL would b a great thing to add